startup_gateway/chat/consumers.py

```
@@ -6,93 +6,111 @@
+
         if not self.user or not self.user.is_authenticated:
             await self.close()
             return
-        
+
         await self.accept()
         self.joined_conversations = set()
-    
+        self.chat = get_chat_service()
+
     async def disconnect(self, close_code):
```

Ruff (0.15.2)

[warning] Unused method argument: `close_code`

(ARG002)